### DISCUSSION TITLE
Definitions for networking $301-$304 on the wiki page. Are they correct?

### DISCUSSION THREAD

**Discussion by jrdarrah** [terjeio/grblHAL#283] [Q&A]
I'm working my way through the $xxx fields to finish configuring the grblhal breakout board and I noticed that what appears to work differ from the information on the wiki page--- 

$3x1=<n> : Interface IP address, default value driver dependent.
IPv4 or IPv6 address, up to 16 octets. May hold IPv4 address as either binary or text.

$3x2=<n> : Interface gateway address, default value driver dependent.
IPv4 or IPv6 address, up to 16 octets. May hold IPv4 address as either binary or text.

$3x3=<n> : Interface netmask, default value driver dependent.
IPv4 or IPv6 address, up to 16 octets. May hold IPv4 address as either binary or text.

$3x3=<n> : Interface netmask, default value driver dependent.
IPv4 or IPv6 address, up to 16 octets. May hold IPv4 address as either binary or text.

$3x4=<n> : IP Mode, default value driver dependent. May be fixed (unchangeable).

0: Static address.
1: DHCP supplied address.
2: Auto IP.

This indicates that the IP mode is $304 

I have this in my grbl definitions

$301 = 0   
$302 = 10.1.1.2   
$303 = 10.1.1.1   
$304 = 255.255.255.0   

When I tried to set $301 to an IP address I get error 3 so I'm guessing that $301 is the IP mode, since it will take 0 for a static IP address, and not $304.  From my raspberry pi I can ping 10.1.1.2. Is the wiki correct?

**Comment by terjeio** [comment]
The [Wiki](https://github.com/terjeio/grblHAL/wiki/Additional-or-extended-settings) is wrong - fixed now. Sorry about that.
